BookingsApi/BookingsApi.UnitTests/Controllers/GetHearingVenuesControllerTest.cs

```diff
@@ -24,14 +23,16 @@ public GetHearingVenuesControllerTest()
     public async Task GetHearingVenueNamesByAllocatedCso_should_return_list_of_hearing_venue_names()
     {
         //Arrange
-        var expectedResponse = new[]{"TestHearingVenueName1", "TestHearingVenueName2"};
+        var venue1 = new HearingVenue(1, "TestHearingVenueName1");
+        var venue2 = new HearingVenue(1, "TestHearingVenueName2");
+        var expectedResponse = new[]{ venue1.Name, venue2.Name };
         var queryResponse = new List<VideoHearing>
         {
             Mock.Of<VideoHearing>(),
             Mock.Of<VideoHearing>()
         };
-        queryResponse[0].HearingVenueName = expectedResponse[0];
-        queryResponse[1].HearingVenueName = expectedResponse[1];
+        queryResponse[0].SetProtected(nameof(HearingVenue), venue1);
```

Comment:
It's not a show-stopper, but would you be against replacing this test with an API. The mocks have been masking a few errors because it doesn't hit all the layers and so the queries or the mappings are off

Reply:
not against it no, although I would like to get these schema changes in to avoid issues. Maybe we can tackle these via a separate tech debt ticket? As there are others that we could probably do with converting to integration tests